deploy.yaml

```
@@ -203,33 +249,34 @@
         oc process -f "{{ openshift_files }}/osbs-registry.yaml"
                    --param-file "{{ openshift_files }}/osbs-box.env"
                    --ignore-unknown-parameters |
-        oc --namespace "{{ registry_namespace }}" apply -f -
+        oc --namespace "{{ registry_namespace }}" apply -f -;
+        oc project osbs-registry;
```

```suggestion
        oc project "{{ registry_namespace }}";
```
And I'd prefer specifying namespace via the `--namespace` param rather than switching projects on the fly, but I don't feel too strongly about it.

---

Shoot! I completely forgot about that. I had so much trouble figuring out the exact incantation, I ended up copying it directly from a terminal session 😂